src/Context/authContext.js

```diff
@@ -0,0 +1,66 @@
+import { createContext, useContext, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { loginAPI, signupAPI } from "../services/authAPI.js";
+import { toast } from 'react-hot-toast';
+
+const AuthContext = createContext();
+
+const AuthProvider = ({children}) => {
+    const navigate = useNavigate();
+    const x = JSON.parse(localStorage.getItem("user"));
```

Comment:
Change this variable name to something relevant.

Reply:
Okay